projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.ts

```diff
@@ -197,15 +196,16 @@ export class NgxLocationPickerComponent implements OnInit, OnDestroy, ControlVal
     return (this.tileLayerType === LeafletTileLayerType.CUSTOM);
   }
 
+  public mapService: MapService = new MapService('browser');
```

Comment:
new keyword niet gebruiken in de component, dit maakt het ontestbaar, injecteer de Mapservice in de constructor!